pytao/interface_commands.py

```diff
@@ -203,14 +215,10 @@ def bunch1(tao, *, ele_id, which="model", ix_bunch="1", coordinate="", verbose=F
         return __execute(tao, cmd, as_dict, method_name='bunch1', cmd_type='real_array')
 
 
-def building_wall_list(tao, ix_section="", *, verbose=False, as_dict=True):
+def building_wall_list(tao, *, ix_section, verbose=False, as_dict=True):
```

Comment:
This one I would expect to be:
```
def building_wall_list(tao, ix_section, *, verbose=False, as_dict=True):
```

Reply:
This is correct:

```
!%% building_wall_list -----------------------
! List of building wall sections or section points
!
! Notes
! -----
! Command syntax:
!   python building_wall_list {ix_section}
! If {ix_section} is not present then a list of building wall sections is given.
! If {ix_section} is present then a list of section points is given
! 
!
! Parameters
! ----------
! ix_section : optional
```

Reply:
Oh, I suppose it needs to be: `ix_section=''`